content/docs/user-guide/project-structure/dvcyaml-files.md

```diff
@@ -76,34 +61,88 @@ Refer to [Visualizing Plots] and `dvc plots show` for more examples.
 
 ### Available configuration fields
 
-- `y` - source from which the Y axis data comes from:
+- `y` - source for the Y axis data:
+
+  - **Top-level plots** (_string, list, dict_):
 
-  - Top-level plots: accepts string, list, or dictionary (like
-    `data_source_path: column/field name`).
+    If plot ID is a path, one or more column/field names is expected. For
+    example:
 
-  - Plot outputs: column/field name found in the source plots file.
+    ```yaml
+    plots:
+      - regression_hist.csv:
+          y: mean_squared_error
+      - classifier_hist.csv:
+          y: [acc, loss]
+    ```
 
-- `x` (string) - source from which the X axis data comes from. An auto-generated
-  _step_ field is used by default.
+    If plot ID is an arbitrary string, a dictionary of file paths mapped to
+    column/field names is expected. For example:
```

Comment:
... it would at least avoid having to explain all this if this then that logic in docs 🙂